examples/lighting-app/nrfconnect/README.md

```diff
@@ -1,289 +1,333 @@
-# CHIP nRF52840 Lighting Example Application
+# CHIP nRF Connect Lighting Example Application
 
-An example application showing the use
-[CHIP](https://github.com/project-chip/connectedhomeip) on the Nordic nRF52840.
+The nRF Connect Lighting Example demonstrates how to remotely control a white non-dimmable
+light bulb.
+
+The example is based on [CHIP](https://github.com/project-chip/connectedhomeip) and the nRF Connect
+platform, and supports remote access and control of a lighting over a low-power, 802.15.4 Thread network.
+It is capable of being paired into an existing CHIP network along with other CHIP-enabled devices.
+
+You can use this example as a reference for creating your own application
```

Comment:
The sentence is a bit confusing to me as it may suggest that buttons are crucial components while they're only used for testing. Maybe we could add a sentence like "LEDs are used to show the state of the device" to the previous paragraph and ignore buttons for now?

Reply:
Let's maybe rewrite the first paragraph:

The nRF Connect Lighting Example demonstrates how to remotely control a white non-dimmable
light bulb. It uses buttons to test changing the lighting and device states and LEDs to show the state of these changes. You can use this example as a reference for creating your own application.